src/lib/loaders/EngineLoader.ts

```diff
@@ -0,0 +1,14 @@
+import {EnginePlugin} from '@salesforce/code-analyzer-engine-api';
+import {SampleEnginePlugin} from '../../tmp/SampleEnginePlugin';
+
+export interface EngineLoader {
+	loadEngines(): EnginePlugin[];
+}
+
+export class EngineLoaderImpl implements EngineLoader {
+	public loadEngines(): EnginePlugin[] {
+		return [
+			new SampleEnginePlugin()
```

Comment:
Since no actual engine plugins exist, we're just using a sample one.

Reply:
I would say that we can just do this in the test side right now and not submit this SampleEnginePlugin at all inside of our src directory.

Reply:
So just import the stub directly from the `test/stubs` folder?

Reply:
No the runtime engine loader (what you are calling EngineLoaderImpl) today would just return [] for now. Then in the tests you would inject a different engineloader with the stubed engine plugins.

Alternatively, we can sprint to get the dynamic loading done - so this can be based off of a config value like engine-plugins: ['absPathToEnginePlugin'].

Then for demo purposes, we could just locally have a config file that loads in some dummy engine plugins.